server/service/src/user_account.rs

```diff
@@ -67,8 +67,9 @@ impl<'a> UserAccountService<'a> {
                 let user_store_repo = UserStoreJoinRowRepository::new(con);
                 let permission_repo = UserPermissionRowRepository::new(con);
 
-                // remove existing user (if exists)
+                // remove all permissions that don't have context
```

Comment:
Having a test that only basic permissions are deleted would be a nice to have!

Reply:
[3777ff1](https://github.com/openmsupply/open-msupply/pull/1345/commits/3777ff1aa8789b588d1062d2d3a69caa94f0da76)